cmd/cli/version/version.go

```diff
@@ -75,10 +78,10 @@ func runVersion(cmd *cobra.Command, oV *VersionOptions) error {
 
 	err := oV.Run(ctx, cmd)
 	if err != nil {
-		return fmt.Errorf("error running version: %w", err)
+		log.Ctx(ctx).Error().Err(err).Msgf("error running version")
 	}
 
-	return err
+	return nil
 }
```

Comment:
I think we still want the `version` command to print an error if it's usual client-server version check doesn't work. Otherwise, a user running `bacalhau version` will just not see any output and not be able to debug why it didn't work.